### DISCUSSION TITLE
How to remove plot protections for the whole server?

### DISCUSSION THREAD

**Discussion by ImCluzzy** [TownyAdvanced/Towny#7468] [Q&A] [answered]
Towny, how to disable privates town and nation?

**Comment by LlmDl** [comment, marked as answer]
https://github.com/TownyAdvanced/Towny/wiki/How-Towny-Works#setting-perms-in-game-with-commands

Does a good job of explaining it.

**Reply by ImCluzzy** [reply]
you didn’t understand me, I need to completely disable privates on the entire server

**Reply by LlmDl** [reply]
Give your players `towny.claimed.*` in their permission nodes.
